src/constant/pages/seo.ts

```diff
@@ -278,6 +278,17 @@ const getSEOMeta = (basePath: any): GetSEOMetaResponseType => {
         'Focus,concentration, The Boring Education, Online Learning, Programming, College Students, Working professional, Career Development, Skill Enhancement, GitHub, Instagram, Twitter, LinkedIn',
       ...seoCommonMeta,
     },
+    
+    [`${basePath}`]: {
```

Comment:
You can't give Basepath here as it's the Param provided by the function. Check how other Dynamic Pages are done